arm/Microsoft.Network/firewallPolicies/ruleGroups/deploy.bicep

```diff
@@ -0,0 +1,41 @@
+@description('Required. Name of the Firewall Policy.')
+param firewallPolicyName string
+
+@description('Required. The name of the rule group to deploy')
+param name string
+
+@description('Required. Priority of the Firewall Policy Rule Group resource.')
+param priority int
+
+@description('Optional. Group of Firewall rules.')
+param rules array = []
+
+@description('Optional. Customer Usage Attribution ID (GUID). This GUID must be previously registered')
+param cuaId string = ''
+
+resource firewallPolicy 'Microsoft.Network/firewallPolicies@2021-03-01' existing = {
+  name: firewallPolicyName
+}
+
+module pid_cuaId '.bicep/nested_cuaId.bicep' = if (!empty(cuaId)) {
```

Comment:
Same comment as before. Let's move that one up

Reply:
done